codegen/smithy-dafny-codegen-cli/src/main/java/software/amazon/polymorph/CodegenCli.java

```diff
@@ -66,13 +72,24 @@ public static void main(String[] args) {
         cliArguments.outputDafnyDir.ifPresent(path -> outputDirs.put(TargetLanguage.DAFNY, path));
         cliArguments.outputJavaDir.ifPresent(path -> outputDirs.put(TargetLanguage.JAVA, path));
         cliArguments.outputDotnetDir.ifPresent(path -> outputDirs.put(TargetLanguage.DOTNET, path));
+        cliArguments.outputGoDir.ifPresent(path -> outputDirs.put(TargetLanguage.GO, path));
+
+        final ServiceShape serviceShape = serviceModel.getServiceShapesWithTrait(LocalServiceTrait.class).stream().findFirst().get();
```

Comment:
Question/Issue/Blocking(?):
If the target service is KMS or DDB, 
then there is no local service,
then this will throw an exception, right?

Also, we now have projects that have multiple local services.
i.e: the MPL, which has KeyStore service and the MPL service.

This change looks very dangerous...
I think we should avoid changing how the serviceShape discovery occurs.

I agree that it should happen here,
but not in this manner.

Instead, I would just move/copy [this line from the CodegenEngine](https://github.com/awslabs/smithy-dafny/blob/main-1.x/codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/CodegenEngine.java#L86):
```
final ServiceShape serviceShape = ModelUtils.serviceFromNamespace(this.model, this.namespace);
```

Reply:
This is for the fast prototype to figure out integration pain points. Now that we have almost everything we need, expect this to be more stable. I changed to use what you suggested for now, but this might evolve the right way in future.